tools/jdbc/src/main/java/org/duckdb/DuckDBConnection.java

```diff
@@ -105,6 +105,12 @@ public void rollback() throws SQLException {
 		}
 	}
 
+	public synchronized void interrupt() throws SQLException {
```

Comment:
Given that this isn't part of the JDBC spec, can you please add a javadoc here documenting what this method does?

Reply:
Yeah it crossed my mind, but it doesn't feel right putting it there?

The underlying c++ interrupt implementation is on the connection object, not on statement. Also, I believe you can only run one query on one connection at a time, so this interrupt is very different from query cancellation mechanism that other database has on a per query/statement basis.

Also, I don't feel very religious about adhering to the standard JDBC spec here because I think it is common for database to have their own jdbc implementation that is different.

Reply:
I will attempt a docstring for this change, but I will be conservative in my wording because I don't fully understand how the underlying c++ interrupt works. this is comparing against sqlite's conn.interrupt docstring which is quite comprehensive and useful.